plugins/services/src/js/components/LogView.js

```diff
@@ -159,8 +152,7 @@ class LogView extends React.Component {
   }
 
   checkIfCloseToTop(container) {
-    const distanceFromTop = DOMUtils.getDistanceFromTop(container);
```

Comment:
We need the `scrollTop` here and not ever the `pageYOffset` (difference in scroll not to the top of the page)